src/super_gradients/training/transforms/pipeline_adaptors.py

```diff
@@ -55,6 +70,18 @@ def prep_for_transforms(self, sample):
             sample = {"image": sample.image, "bboxes": sample.bboxes_xyxy, "labels": sample.labels, "is_crowd": sample.is_crowd}
         elif self.sample_type == SampleType.SEGMENTATION:
             sample = {"image": np.array(sample.image), "mask": np.array(sample.mask)}
+        elif self.sample_type == SampleType.POSE_ESTIMATION:
+
+            bboxes_xyxy = xywh_to_xyxy(bboxes=np.array(sample.bboxes_xywh), image_shape=sample.image.shape)
+
+            sample = {
+                "image": sample.image,
+                "bboxes": bboxes_xyxy,
+                "labels": np.zeros(sample.bboxes_xywh.shape[0]),  # Dummy value, this is required for Albumentation. Here, all classes are the same.
+                "mask": np.array(sample.mask),
+                "is_crowd": sample.is_crowd,
+                "keypoints": sample.joints,
```

Comment:
I think we should also pass keypoint_params to A.Compose to indicate format of keypoints (XY).
Also, we have to ensure a few important aspects:
- That transforms can handle (x,y,visibility) format (visibility should not be changed by Albumentations)
- We should update visibility status of each keypoint if it goes outside the image bounds. Ideally we should check that after each transform.
- That we don't remove keypoints (When doing a crop for instance).
- That order of keypoints does not change during transforms. (The index of keypoint mark it's joint so order of keypoint should be fixes)

Reply:
- When a joint is out of image, we set to (0, 0, 0)
- When a skeleton is full of (0,0,0) do we remove the full skeleton?
- Or, do we drop it only when the bbox is removed ? Or both ?

Reply:
This commit covers it all, but I'm not sure about the skeleton filtering. 
[2c3d0b7](https://github.com/Deci-AI/super-gradients/pull/1710/commits/2c3d0b79b875b3a102c3df18e43c4891c5c5f35e)

Reply:
I added `sample_sanitize` here 65345a579331e52c8fff35873bfe5fb3fd126e74